src/test/lit-html_test.ts

```diff
@@ -366,7 +354,9 @@ suite('lit-html', () => {
               ${'baz'}
               <p>${'qux'}</p>
             </div>`, container);
-        assert.equal(container.innerHTML, `<div foo="bar">baz<p>qux</p></div>`);
+        assert.equal(container.innerHTML, `<div foo="bar">
+              baz
+              <p>qux</p></div>`);
```

Comment:
The name of this test could stand to be improved. It is a fair - albeit vague - description of what is being tested, but there is no reason why "stuff" _must_ include significant whitespace. A better test name will help to preserve such details as the test changes over time.

Reply:
It may even make sense to break this into a couple of tests that each focus on a particular detail of rendering complex "stuff." For example, one test that focuses on nesting and one that focuses on multiline significant whitespace.

Reply:
I completely agree, and this test _might_ be obsolete by now, but at the time I wrote it I had very little coverage of specific scenarios that this was catching. This was serving as a canary for situations I didn't know to test yet.

Since adding it I've tried to add a new test for every time I break this one, which is basically your second comment. I think for a better version of this kind of coverage we should add some larger real-world-style templates and goldens.